blocks/editable/style.scss

```diff
@@ -56,6 +56,19 @@
 	}
 }
 
+.editor-visual-editor__block:not( .is-selected ) .blocks-editable.drop-cap-true {
```

Comment:
Should we move this style into the `text` block stylesheet instead? (or create one)